### PR TITLE
EZP-31241: Changed behaviour for showing not checked value in checkbox

### PR DESCRIPTION
**Pull request by GrabowskiM**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31241
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Show 'No' in content view when checkbox is empty.

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/content/content_view_fields.html.twig [hunk 1]
```diff
@@ -19,7 +19,9 @@
                                 <div class="ez-content-field">
                                     <p class="ez-content-field-name">{{ fieldDefinition.name }}:</p>
                                     <div class="ez-content-field-value">
-                                        {% if ez_is_field_empty(content, fieldDefinition.identifier) %}
+                                        {% if ez_is_field_empty(content, fieldDefinition.identifier) and fieldDefinition.fieldTypeIdentifier is same as('ezboolean') %}
+                                            <em>{{ 'ezboolean.no'|trans|desc('No') }}</em>
```
Comment (dew326) [inline src/bundle/Resources/views/content/content_view_fields.html.twig]: I'm not sure if this is the way. I think in the case of ezboolean the field should be rendered even if it's empty.

**Comment by katarzynazawada** [comment]
Now there's small inconsistency - **No** (with capital letter) and **yes** (with lowercase letter)

**Comment by GrabowskiM** [comment]
> Now there's small inconsistency - **No** (with capital letter) and **yes** (with lowercase letter)

That's inconsistency that we have in whole system, as these are translation from xliff file: https://github.com/ezsystems/ezplatform-admin-ui/blob/1.5/src/bundle/Resources/translations/fieldtypes_preview.en.xliff#L10
As I see, it's only in 2.5 version (1.5 of module), as it was fixed in 3.0: https://github.com/ezsystems/ezplatform-admin-ui/pull/1115/files#diff-63de7c5f52281012572e3a2b3685d538
So @DominikaK , should I change it in translation file? In theory it will affect whole system.

**Comment by DominikaK** [comment]
True, the recent changes in labels were introduced in 3.0 only, so I don't think the change is necessary

**Comment by lserwatka** [comment]
@GrabowskiM could you merge it up?

**Pull request merged**